rfc_x86_inductor_quantizer.md

```diff
@@ -0,0 +1,343 @@
+## Feature
+This RFC proposes to add [TorchInductor](https://dev-discuss.pytorch.org/t/torchinductor-a-pytorch-native-compiler-with-define-by-run-ir-and-symbolic-shapes/747) as one of the backends for [Quantization 2.0 in Export](https://pytorch.org/tutorials/prototype/quantization_in_pytorch_2_0_export_tutorial.html).
```

Comment:
I would suggest we narrow the scope of the RFC to x86 CPU while leave some notes as open discussions on how quantization of other devices can be supported with the inductor backend. For different devices, different quantizers have to be added while we should still be able to lower the quantized graph via inductor compiler. A question is how ARM cpu can be supported. We can probably still reuse the quantizer proposed here and leverage oneDNN/ARM as the backend.

Reply:
Thanks and changed.